### ISSUE TITLE
add webpage

### ISSUE THREAD

**Issue by dankelley** [ArgoCanada/argoFloats#63]
As discussed in an f2f today, @richardsc has volunteered to do this.

**Comment by dankelley** [comment]
@j-harbin and I discussed how this website could be set up with gh-pages, during our video-creation meeting today.  Since I have experience with jekyll, I'll set up the skeleton.

This has got to the "dog food" stage, because I could not for the life of me find our first video by searching within youtube.  Having it listed in our webpage thus help not just users, but at least one of the developers 😆

**Comment by richardsc** [comment]
I was hoping to take a poke at this, actually. If you've already got started, no worries, but I have been looking at some of the different jekyll themes and was also considering trying out [hugo](https://gohugo.io/) which I've heard is better than jekyll.

**Comment by dankelley** [comment]
Oh, it would be great if you could look into that, @richardsc.  I've not started.

**Comment by dankelley** [comment]
I was just looking at the 'covdata' package, discussed in an interesting blog posting (https://kieranhealy.org/blog/archives/2020/04/23/apples-covid-mobility-data/), and see that the very nice webpage for covdata (https://kjhealy.github.io/covdata/index.html) is made with pkgdown.  That might be a nice option since it is designed for packages, and so has great things like the function manpages.

**Comment by richardsc** [comment]
Oh, this is a good idea. I started going down the "basic jekyll site" rabbit hole last night (essentially using the template I have for my blog at clarkrichards.org) and got a little frustrated with all the jekyll dependencies ...

**Comment by richardsc** [comment]
@dankelley -- I'm playing with `pkgdown`, but I noticed that you already had a `docs/` folder in the repo. It appeared to be a holdover from when things were copied from the old `dc` repo (based on the commit notes), so I am just going to wipe it out.

The problem is that is where `pkgdown` puts the website it generates, and it the existing one conflicts.

**Comment by dankelley** [comment]
Right, you should wipe whatever is in `docs` at the moment.

**Comment by richardsc** [comment]
👍

**Comment by richardsc** [comment]
Hm. Something seems to be wrong ... If I go to the page at

https://argocanada.github.io/argoFloats/index.html

I see what looks like Dan's "old" website, not what pkgdown was supposed to create ...

![image](https://user-images.githubusercontent.com/233584/80243985-94c11e80-863e-11ea-98a8-08a5a265d2a3.png)

**Comment by richardsc** [comment]
I wonder if it just takes some time for the new website sources to propagate into the Github pages system?

**Comment by dankelley** [comment]
Oh, yes, it takes a while.  I think it gets faster if you do an update. Maybe wait 15 minutes and see, but likely after that it might update in just a few minutes.  I suppose this is server-load dependent.

**Comment by dankelley** [comment]
Hm. It's been 1/2 an hour and still no update.  As I remember, if there are any problems, then it will not update the view.  Possibly it emails an error message, although I don't remember that ... this must be documented on the GH site somewhere.  

I don't see anything from GH in my spam mailbox.  Maybe those will go to the email set by ArgoCanada?

My recollection is that it's not fun debugging GH webpages.  Luckily, when it does work, it should just be html that will continue working (well, if the javascript in it doesn't fail with an update...)

**Comment by richardsc** [comment]
Yeah, something seems to be wrong. I'll keep poking at it

**Comment by richardsc** [comment]
I think I might have figured it out (I hope).

The problem was that based on the `pkgdown` documentation I had set the gh-pages source as `docs/` on the "master" branch, e.g.
![image](https://user-images.githubusercontent.com/233584/80265033-825dd980-866c-11ea-922c-7d2f8f34f059.png)

But of course, we work off `develop`, so the changes weren't being reflected there and was simply showing the "old" website (that was ported over from `dc`).

So, I merged develop into master, and deleted the gh-pages branch from my jekyll tests last night, and hopefully when it builds the site all will be good to go. 🤞
